### PR TITLE
Additional JSON fields for logged exceptions

### PR DESCRIPTION
**Pull request by Channyboy**
#build
Related to #17559 
Beta Guarded

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build #libby

**Comment by Channyboy** [comment]
Failures are not related to this PB. Failures are UI related.

**Comment by Channyboy** [comment]
#build #libby

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java [hunk 1]
```diff
@@ -199,7 +246,6 @@ private void checkJsonMessage(String line, ArrayList<String> mandatoryKeyList, A
                 value = "" + jsonObj.get(key);
                 Log.finer(c, method, "key=" + key + ", value=" + value);
             } else if (optionalKeyList.contains(key)) {
-                optionalKeyList.remove(key);
```
Comment (Channyboy) [inline dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java:202]: This did nothing, so was removed.

File: dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java [hunk 2]
```diff
@@ -211,11 +257,11 @@ private void checkJsonMessage(String line, ArrayList<String> mandatoryKeyList, A
         }
         if (mandatoryKeyList.size() > 0) {
             Log.info(c, method, "Mandatory keys missing: " + mandatoryKeyList.toString());
-            Assert.fail("Mandatory FFDC keys missing: " + mandatoryKeyList.toString() + ". Actual JSON was: " + line);
+            Assert.fail("Mandatory keys missing: " + mandatoryKeyList.toString() + ". Actual JSON was: " + line);
         }
         if (invalidFields.size() > 0) {
             Log.info(c, method, "Invalid keys found: " + invalidFields.toString());
-            Assert.fail("Invalid FFDC keys found: " + invalidFields.toString() + ". Actual JSON was: " + line);
```
Comment (Channyboy) [inline dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java:218]: This and above were probably copy and pasted before. This method isn't specific to FFDC keys.

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by Channyboy** [comment]
#libby

**Comment by Channyboy** [comment]
#build

**Pull request merged**